share/completions/apropos.fish

```diff
@@ -1,21 +1,29 @@
-
 function __fish_complete_apropos
     if test (commandline -ct)
         set str (commandline -ct)
-        apropos $str 2>/dev/null | sed -e "s/^\(.*$str\([^ ]*\).*\)\$/$str\2"\t"\1/"
+        switch $str
```

Comment:
So wait, this is meant to stop it from passing arguments starting with `-` to apropos?

Wouldn't `apropos -- $str` work for that?

Reply:
To supress completions for text starting with "-" as it means the actual apropos option completions won't show:

https://user-images.githubusercontent.com/20397027/77248543-1eda2980-6c32-11ea-9a67-db33c0886376.png

Reply:
Technically I think it's possible for a manpage name to start with "-". But since I don't think I've ever seen one that does, this is probably alright.